src/cmd/cli/command/commands.go

```diff
@@ -1059,6 +1060,11 @@ func configureLoader(cmd *cobra.Command) *compose.Loader {
 	if err != nil {
 		panic(err)
 	}
+	// Avoid common mistakes: using -p with a provider name instead of -P
+	var prov cliClient.ProviderID
+	if prov.Set(projectName) == nil {
+		term.Warnf("Did you mean to use -P%s instead of -p?", projectName)
```

Comment:
Should we offer users a Yes/No choice? If it's just a warning the users won't be able to stop the current deployment to fix this.

Reply:
good idea